spec/ics-024-host-requirements/README.md

```diff
@@ -99,11 +99,14 @@ Parts of the private store MAY safely be used for other purposes as long as the
 Keys used in the private store MAY safely vary as long as there exists a bipartite mapping between the key formats defined herein and the ones
 actually used in the private store implementation.
 
+Note that the client-related paths listed below reflect the Tendermint client as defined in [ICS 7](../ics-007-tendermint-client) and may vary for other client types.
+
 | Store          | Path format                                                                    | Value type        | Defined in |
 | -------------- | ------------------------------------------------------------------------------ | ----------------- | ---------------------- |
-| privateStore   | "clients/{identifier}"                                                         | ClientState       | [ICS 2](../ics-002-client-semantics) |
-| provableStore  | "clients/{identifier}/consensusState"                                          | ConsensusState    | [ICS 2](../ics-002-client-semantics) |
 | provableStore  | "clients/{identifier}/type"                                                    | ClientType        | [ICS 2](../ics-002-client-semantics) |
+| privateStore   | "clients/{identifier}"                                                         | ClientState       | [ICS 2](../ics-007-tendermint-client) |
+| provableStore  | "clients/{identifier}/consensusStates/{height}"                                | ConsensusState    | [ICS 7](../ics-007-tendermint-client) |
+| privateStore   | "clients/{identifier}/connections                                              | []Identifier      | [ICS 3](../ics-003-connection-semantics) |
```

Comment:
what is this path for?

Reply:
It's used for reverse lookup of what connections are using a client.